ansible/molecule/bootstrap/molecule.yml

```diff
@@ -0,0 +1,21 @@
+---
+#
+# Copyright (C) 2021 Julien Lecomte <julien@lecomte.at>
+# Copyright (C) 2021 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-only
+#
+
+platforms:
+  - name: debops-apt-stretch
+    image: registry.gitlab.com/dockerio/debian:stretch
```

Comment:
The docker hub rate limits are easily encountered when all tests "no deps" test are molecule based,